teacher.rb

```diff
@@ -3,7 +3,7 @@
 class Teacher < Person
   attr_accessor :specialization
 
-  def initialize(age, specialization, name = 'Unknown', parent_permission: true)
```

Comment:
[OPTIONAL] I would suggest not to remove default values from parameters. It will be helpful for you to handle errors. If someone enters wrong value or leave it empty in name part,  it will show you the `Unknown` message in name part.